app/src/main/java/com/github/se/cyrcle/ui/review/ReviewScreen.kt

```diff
@@ -72,27 +76,27 @@ fun ReviewScreen(
   }
   var textValue by remember { mutableStateOf(if (ownerHasReviewed) matchingReview?.text!! else "") }
 
-  val context = LocalContext.current // Get the current context
+  val context = LocalContext.current
 
   Scaffold(
       topBar = {
         TopAppBar(
             navigationActions = navigationActions,
-            if (ownerHasReviewed) stringResource(R.string.review_screen_title_edit_review)
-            else stringResource(R.string.review_screen_title_new_review))
+            title =
+                if (ownerHasReviewed) stringResource(R.string.review_screen_title_edit_review)
+                else stringResource(R.string.review_screen_title_new_review))
       }) { paddingValues ->
         Column(
             modifier = Modifier.fillMaxSize().padding(paddingValues).padding(16.dp),
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.Center) {
-              ScoreStars(sliderValue.toDouble(), scale = 2.5f)
-              // Display the experience text based on the slider value
+              ScoreStars(sliderValue.toDouble(), scale = scaleFactor) // Use dynamic scale factor
               Text(
                   text =
                       when (sliderValue) {
-                        in 1f..2f -> stringResource(R.string.review_screen_poor_review)
-                        in 2f..3f -> stringResource(R.string.review_screen_bad_review)
-                        3f -> stringResource(R.string.review_screen_average_review)
+                        in 0f..1f -> stringResource(R.string.review_screen_terrible_review)
+                        in 1f..2f -> stringResource(R.string.review_screen_bad_review)
+                        in 2f..3f -> stringResource(R.string.review_screen_average_review)
```

Comment:
Just for consistency but you define the range from 1 to 2 , 2 to 3 and then 3.5 to 4 maybe you could change that but this is not up to your PR though so it is not blocking.

Reply:
I can address this now

Reply:
Addressed in my most recent commit!

Reply:
Thanks !